server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRToAuditDownload.java

```diff
@@ -236,7 +235,7 @@ public String endpointBody(final Request the_request, final Response the_respons
       }
      
       response_list.addAll(BallotSelection.toResponseList(cvr_to_audit_list));
-      response_list.sort(new BallotOrderComparator());
+      response_list.sort(null);
```

Comment:
This looks weird but it's necessary - this is defined to use the natural ordering (`Comparable` implementations) if `null` is passed, but there is no 0-arity version of `sort`.